lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -470,40 +470,39 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
                   return success();
                 });
   patterns.onOp(
-    "Scatter", 9,
-    [](OpBinder binder, ConversionPatternRewriter &rewriter) {
-      int64_t axis;
-      if (binder.s64IntegerAttr(axis, "axis", {}))
-        return rewriter.notifyMatchFailure(binder.op, "axis bind failure");
-
-      Torch::ValueTensorType resultTy;
-      Value data, indices, updates;
-      if (binder.tensorOperandAtIndex(data, 0) || 
-          binder.tensorOperandAtIndex(indices, 1) || 
-          binder.tensorOperandAtIndex(updates, 2) || 
-          binder.tensorResultType(resultTy))
-        return failure();
+      "Scatter", 9, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
```

Comment:
Huh... Is this just from clang-format? I wonder why this was causing an issue in the CI just for this PR.

Reply:
See #3407 

I think it might be a good idea to sync this branch with main once that PR gets merged instead of introducing these changes here.

Reply:
Done